samcli/commands/deploy/command.py

```diff
@@ -150,11 +151,14 @@
     "the output AWS CloudFormation template. YAML is used by default.",
 )
 @click.option(
-    "--resolve-s3",
+    "--resolve-s3", required=False, is_flag=True, help="Automatically resolve S3 bucket for non-guided deployments."
+)
+@click.option(
+    "--resolve-image-repos",
     required=False,
     is_flag=True,
-    help="Automatically resolve s3 bucket for non-guided deployments."
-    "Do not use --s3-guided parameter with this option.",
+    help="Automatically create and delete ECR repositories for image-based functions in non-guided deployments. "
```

Comment:
should we talk about the mapping here? on new ecr repos per function resource in the template.

Reply:
Added more details on mapping.